packages/react/src/components/HelpText/HelpText.stories.tsx

```diff
@@ -13,24 +13,15 @@ const decorators = [
 ];
 
 export default {
-  title: 'Komponenter/HelpText',
+  'aria-label': 'Komponenter/HelpText',
```

Comment:
This title needs to be here to place the component correctly in storybook
![image](https://github.com/user-attachments/assets/a2c74fef-b8b2-4147-8b35-9e6469af1dea)
```suggestion
  title: 'Komponenter/HelpText',
  'aria-label': 'Komponenter/HelpText',
```

Reply:
Oh 🙈